src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,77 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+import { PaginationParams } from '../../types';
+import React from 'react';
+
+export const Pagination: React.FC<PaginationParams> = ({
+  total,
+  perPage,
+  currentPage = 1,
+  onPageChange,
+}) => {
+  const numOfPages = Math.ceil(total / perPage);
+
+  const pageClickHandler = (i: number) => {
+    if (currentPage !== i + 1) {
+      onPageChange(i + 1);
+    }
+  };
+
+  return (
+    <>
+      <ul className="pagination">
+        <li
+          className={cn('page-item', {
+            disabled: currentPage === 1,
+          })}
+        >
+          <a
+            data-cy="prevLink"
+            className="page-link"
+            href="#"
```

Comment:
Using `href="#"` can cause the page to jump to the top when clicked. Consider using `event.preventDefault()` in the `onClick` handler to prevent this default behavior.